docs/integrations/microsoft-azure/web-apps.md

```diff
@@ -13,11 +13,24 @@ The Azure Web Apps app allows you to collect Azure web server and application di
 
 For more information, see [Azure Web Apps](https://azure.microsoft.com/en-us/services/app-service/web/).
 
-## Log types
+## Log and metric types
 
-The Azure Web Apps app supports:
-* **Web Server Logging.** Information about HTTP transactions using the [W3C extended log file format](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx). This is useful when determining overall site metrics such as the number of requests handled or how many requests are from a specific IP address.
-* **Application Diagnostics Logs.** Application diagnostics allows you to capture information produced by a web application. ASP.NET applications can use the [System.Diagnostics.Trace](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx) class to log information to the application diagnostics log.
+For Azure Web Apps, you can collect the following logs and metrics:
+
+- **Resource logs**, which provide an insight into operations that were performed within an Azure resource. For a complete schema for resource logs refer to the below documentation:
+    * [Web Server Logging](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx)
+    * [Application Diagnostics Logs](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx)
+    * [Activity logs](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema)
+    * [AppServiceEnvironmentPlatformLogs](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/appserviceenvironmentplatformlogs)
```

Comment:
serviceenvironplatformlogs I think these do not come for web apps?

Reply:
Removed